assets/tests/components/propertiesPanel/crowdingDiagram.tippy.test.tsx

```diff
@@ -0,0 +1,31 @@
+import "@testing-library/jest-dom"
+import { render, screen } from "@testing-library/react"
+import userEvent from "@testing-library/user-event"
+import React from "react"
+import CrowdingDiagram from "../../../src/components/propertiesPanel/crowdingDiagram"
+import { Crowding } from "../../../src/models/crowding"
+import crowdingFactory from "../../factories/crowdingFactory"
+import { mockFullStoryEvent } from "../../testHelpers/mockHelpers"
+
+// Tippy sets some accessability attributes that is used for testing
+// which means it's better to not mock Tippy
+// Due to this unmock interacting badly with other tests
+jest.unmock("@tippyjs/react")
```

Comment:
I'm not having any luck with that. I've read some stuff that says jest uses hoisting to make it's mock calls happen before imports, and doing any unmocking seems to require hoisting, which doesn't work(for me, so far) if it's invoked within a describe/test/or other function.

because tippy is mocked for the entire application, my guess/observation is that the unmock needs to happen at the root of the module so it gets hoisted properly.

I could possibly unmock at the root, and then mock it for the regular tests using beforeAll/afterAll?
Any preferences? I'm leaning toward leaving it because it's easier.

Reply:
Yeah that is more of a pain than I thought it would be - leaving it seems good to me 😃